extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -314,7 +314,8 @@ export class AmpStoryPage extends AMP.BaseElement {
 
   /** @private */
   markPageAsLoaded_() {
-    dispatch(this.element, EventType.PAGE_LOADED, true);
+    dispatch(this.win, this.element, EventType.PAGE_LOADED, undefined,
```

Comment:
nit: when you have an ambiguous literal argument that doesn't explain what it means semantically (in this case `undefined`, but is equally true for literals like `true`, `17`, `"foo"`, etc), it's nice to leave a comment inline that explains what that argument represents, like:

```javascript
dispatch(this.win, this.element, EventType.PAGE_LOADED,
    /* payload */ undefined, {bubbles: true});
```

Reply:
ahh good tip! thanks :)